### ISSUE TITLE
[Feature request] LoRA support

### ISSUE THREAD

**Issue by OKUA1** [ngxson/wllama#105]
Hi, 

The most recent approach of llama.cpp handling the LoRA without merging the weights and allowing for hot-swappable adapters sounds very interesting for in-browser use cases. Are there any plans to implement it into wllama ?

**Comment by bil-ash** [comment]
Yes, LoRA support would be a very nice thing in browser and would help to keep resource usage limited in browser. If you can provide the ability to load a base model, then load a lora, use it, then unload it and then use another lora, that would be the best approach

**Comment by nfplay** [comment]
This would be a killer feature, any chance of a timeline? Thanks!
